cpp/harvest/nacjd.cc

```diff
@@ -343,9 +353,15 @@ void ParseJSONAndWriteMARC(MARC::Writer * const title_writer) {
 
 
 int Main(int argc, char *argv[]) {
-    if (argc != 3)
+    if (argc < 3 or argc > 4)
         Usage();
 
+ 
+    int count_rows = 9;
```

Comment:
1) before the change, count_rows has been hardcoded with the value 9000, so we might want to keep that value
2) Please always use the constructor notation when initializing variables: `int count_rows(9);`